byway/src/main/java/com/google/sps/servlets/DestinationsServlet.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Key;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/api/destinations")
+public class DestinationsServlet extends HttpServlet {
+
+    private final static class UserLocations{ 
+      private String start;
+      private ArrayList<String> destinations;
+
+        public UserLocations(String start, ArrayList destinations) {
+            this.start = start;
+            this.destinations = destinations;
+        }
+    };
+
+private final UserLocations places = new UserLocations("", new ArrayList<String>());
```

Comment:
This doesn't seem to be doing much at the moment. You may just want to get rid of it.

Reply:
This array is storing the destinations that a user adds, and then the whole array is sent to datastore under one propery

Reply:
I see, this is acceptable for now to get some basic functionality.